packages/components-library/src/media-selector/index.tsx

```diff
@@ -159,6 +168,10 @@ const MediaSelector = (props: MediaSelectorProps) => {
         }
     };
 
+    console.log("selectedFile", selectedFile);
```

Comment:
There are console.log in the commit. Please remove.

Reply:
Removed